app/src/main/java/com/example/tasky/itemdetail/presentation/components/TimeDatePicker.kt

```diff
@@ -35,83 +36,86 @@ fun TimeDatePicker(
     onDateClicked: () -> Unit,
     modifier: Modifier = Modifier
 ) {
-    Row(
-        modifier = modifier
-            .height(70.dp)
-            .fillMaxWidth()
-            .padding(horizontal = 16.dp),
-        verticalAlignment = Alignment.CenterVertically,
-        horizontalArrangement = Arrangement.SpaceBetween
-    ) {
-
+    Column {
         Row(
-            modifier = Modifier
-                .clickable(enabled = isEditMode) { onTimeClicked() }
+            modifier = modifier
+                .height(70.dp)
+                .fillMaxWidth()
+                .padding(horizontal = 16.dp),
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.SpaceBetween
         ) {
-            Text(
+
+            Row(
                 modifier = Modifier
-                    .padding(start = 6.dp),
-                text = timePrefix,
-                color = Black,
-                fontSize = 16.sp
-            )
+                    .clickable(enabled = isEditMode) { onTimeClicked() }
+            ) {
+                Text(
+                    modifier = Modifier
+                        .padding(start = 6.dp),
+                    text = timePrefix,
+                    color = Black,
+                    fontSize = 16.sp
+                )
 
-            if (timePrefix == "From") {
-                Spacer(modifier = Modifier.width(17.dp))
-            } else {
-                Spacer(modifier = Modifier.width(37.dp))
+                if (timePrefix == "From") {
```

Comment:
Mmm not fan of this. Since this wont scale if you add localization

What if you use Modifier.weight on the Row?